dex/testing/eth/harness.sh

```diff
@@ -107,19 +130,64 @@ tmux send-keys -t $SESSION:0 "cd ${NODES_ROOT}/harness-ctl" C-m
 
 echo "Starting simnet alpha node"
 "${HARNESS_DIR}/create-node.sh" "$SESSION:1" "alpha" "$ALPHA_NODE_PORT" \
-	"$CHAIN_ADDRESS" "$PASSWORD" "$CHAIN_ADDRESS_JSON " \
-	"$CHAIN_ADDRESS_JSON_FILE_NAME" "$ALPHA_ADDRESS" "$PASSWORD" \
-	"$ALPHA_ADDRESS_JSON" "$ALPHA_ADDRESS_JSON_FILE_NAME" "$ALPHA_NODE_KEY"
+	"$CHAIN_ADDRESS" "$PASSWORD" "$CHAIN_ADDRESS_JSON" \
+	"$CHAIN_ADDRESS_JSON_FILE_NAME" "$ALPHA_ADDRESS_JSON" "$ALPHA_ADDRESS_JSON_FILE_NAME" \
+	"$ALPHA_NODE_KEY" "fast"
 
 echo "Starting simnet beta node"
 "${HARNESS_DIR}/create-node.sh" "$SESSION:2" "beta" "$BETA_NODE_PORT" \
-	"$CHAIN_ADDRESS" "$PASSWORD" "$CHAIN_ADDRESS_JSON " \
-	"$CHAIN_ADDRESS_JSON_FILE_NAME" "$BETA_ADDRESS" "$PASSWORD" \
-	"$BETA_ADDRESS_JSON" "$BETA_ADDRESS_JSON_FILE_NAME" "$BETA_NODE_KEY"
+	"$CHAIN_ADDRESS" "$PASSWORD" "$CHAIN_ADDRESS_JSON" \
+	"$CHAIN_ADDRESS_JSON_FILE_NAME" "$BETA_ADDRESS_JSON" "$BETA_ADDRESS_JSON_FILE_NAME" \
+	"$BETA_NODE_KEY" "fast"
+
+echo "Starting simnet gamma node"
+"${HARNESS_DIR}/create-node.sh" "$SESSION:3" "gamma" "$GAMMA_NODE_PORT" \
+	"_" "_" "_" "_" "$GAMMA_ADDRESS_JSON" "$GAMMA_ADDRESS_JSON_FILE_NAME" \
+	"$GAMMA_NODE_KEY" "light"
+
+echo "Starting simnet delta node"
+"${HARNESS_DIR}/create-node.sh" "$SESSION:4" "delta" "$DELTA_NODE_PORT" \
+	"_" "_" "_" "_" "$DELTA_ADDRESS_JSON" "$DELTA_ADDRESS_JSON_FILE_NAME" \
+	"$DELTA_NODE_KEY" "light"
+
+sleep 1
 
-# NOTE: This will cause beta to connect automatically to alpha.
+# NOTE: Connecting a node will add for both. Also, light nodes take longer to
+# set up. They will show 0 peers for some amount of time even after adding here.
 echo "Connecting nodes"
 "${NODES_ROOT}/harness-ctl/alpha" "attach --exec admin.addPeer('enode://${BETA_ENODE}@127.0.0.1:$BETA_NODE_PORT')"
+"${NODES_ROOT}/harness-ctl/alpha" "attach --exec admin.addPeer('enode://${GAMMA_ENODE}@127.0.0.1:$GAMMA_NODE_PORT')"
+"${NODES_ROOT}/harness-ctl/alpha" "attach --exec admin.addPeer('enode://${DELTA_ENODE}@127.0.0.1:$DELTA_NODE_PORT')"
+"${NODES_ROOT}/harness-ctl/beta" "attach --exec admin.addPeer('enode://${GAMMA_ENODE}@127.0.0.1:$GAMMA_NODE_PORT')"
+"${NODES_ROOT}/harness-ctl/beta" "attach --exec admin.addPeer('enode://${DELTA_ENODE}@127.0.0.1:$DELTA_NODE_PORT')"
+"${NODES_ROOT}/harness-ctl/gamma" "attach --exec admin.addPeer('enode://${DELTA_ENODE}@127.0.0.1:$DELTA_NODE_PORT')"
+
+sleep 1
+
+SEND_AMT=5000000000000000000000
+echo "Sending 5000 eth to beta and gamma."
+"${NODES_ROOT}/harness-ctl/alpha" "attach --preload ${NODES_ROOT}/harness-ctl/send.js --exec send(\"${ALPHA_ADDRESS}\",\"${GAMMA_ADDRESS}\",${SEND_AMT})"
+"${NODES_ROOT}/harness-ctl/alpha" "attach --preload ${NODES_ROOT}/harness-ctl/send.js --exec send(\"${ALPHA_ADDRESS}\",\"${DELTA_ADDRESS}\",${SEND_AMT})"
+
+# TODO: After we start mining blocks on alpha, it seems that we can never mine
+# blocks on beta. Doing so sends alpha into some sort of error mode. Find out
+# why that is and correct it if a settings miss.
+#
```

Comment:
It looks like this is no longer the case on geth `v1.10.1` so was probably due to a bug. Will remove this todo.